frontend/src/commit-panel/Notice.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+import { DetailsLevel } from '../enums/enums';
+
+interface CommitPanelNoticeProps {
+  detailsLevel: DetailsLevel;
+  onDetailsLevelChange(detailsLevel: DetailsLevel): void;
+}
+
+export default class Notice extends React.Component<CommitPanelNoticeProps, {}> {
+
+  onDetailsClick = (e: React.MouseEvent) => {
+    e.preventDefault();
+
+    const { detailsLevel, onDetailsLevelChange } = this.props;
+    onDetailsLevelChange(detailsLevel === DetailsLevel.None ? DetailsLevel.Overview : DetailsLevel.None);
+  };
+
+  render() {
+    const { detailsLevel } = this.props;
+
+    return (
+      <p>
+        {`You have `}
+        <a
+          href='http://docs.versionpress.net/en/feature-focus/undo-and-rollback#uncommitted-files'
+          target='_blank'
+        >
+          uncommitted changes
```

Comment:
If you **really** insist on this style, then I can make a compromise and use it. But personally I'd prefer mine, I believe that is cleaner and more readable, at least for me.